strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/PypiArtifactGenerator.java

```diff
@@ -80,6 +89,12 @@ public PypiArtifactGenerator(String basedir)
         this.basedir = Paths.get(basedir);
     }
 
+    @Override
+    public void setLicenses(LicenseConfiguration[] licenses)
+    {
+        this.licenses = licenses;
```

Comment:
This is where you should be checking if there is just one specified license, not in the `getLicenseName()` method below.

Reply:
Done.

Reply:
Thanks! :+1: